stubs/pycountry/pycountry/db.pyi

```diff
@@ -0,0 +1,43 @@
+import logging
+from collections.abc import Callable, Iterator
+from typing import Any, TypeVar
+
+logger: logging.Logger
+
+_F = TypeVar("_F", bound=Callable[..., Any])
+
+class Data:
+    def __init__(self, **fields: str) -> None: ...
+    def __getattr__(self, key: str) -> str: ...
+    def __setattr__(self, key: str, value: str) -> None: ...
+    def __dir__(self) -> list[str]: ...
+    def __iter__(self) -> Iterator[tuple[str, str]]: ...
+
+class Country(Data): ...
+class Subdivision(Data): ...
+
+def lazy_load(f: _F) -> _F: ...
+
+class Database:
+    data_class: type | str
+    root_key: str | None
+    no_index: list[str]
+    filename: str
+    factory: type
+    objects: list[Data]
+    index_names: set[str]
+    indices: dict[str, dict[str, Data]]
+
+    def __init__(self, filename: str) -> None: ...
+    @lazy_load
+    def add_entry(self, **kw: str) -> None: ...
+    @lazy_load
+    def remove_entry(self, **kw: Any) -> None: ...
+    @lazy_load
+    def __iter__(self) -> Iterator[Data]: ...
+    @lazy_load
+    def __len__(self) -> int: ...
+    @lazy_load
+    def get(self, *, default: Data | None = ..., **kw: Any) -> Data | None: ...
```

Comment:
There is a [check inside](https://github.com/pycountry/pycountry/blob/6c2ac46a3e0e475031261e98d76af806bdabc2ca/src/pycountry/db.py#L170) that `kw` should be `str`, so I don't see any reason to set it to `Any`.
```suggestion
    def get(self, *, default: Data | None = ..., **kw: str) -> Data | None: ...
```